README.md

```diff
@@ -1,5 +1,6 @@
 # Emeralddb
-[![.github/workflows/build.yml](https://github.com/Rembrant777/db-engine/actions/workflows/build.yml/badge.svg?branch=db_init)](https://github.com/Rembrant777/db-engine/actions/workflows/build.yml)
```

Comment:
To many workflow failures will cause the badge's status keeps failing, remove no use workflow history can resolve this issue.